src/main/java/io/cdap/plugin/dlp/configs/DlpTransformConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.cdap.plugin.dlp.configs;
+
+import com.google.privacy.dlp.v2.PrimitiveTransformation;
+import io.cdap.cdap.api.data.schema.Schema;
+import io.cdap.cdap.etl.api.FailureCollector;
+
+import java.util.List;
+
+/**
+ *
+ */
+public interface DlpTransformConfig {
```

Comment:
why is this an interface? Do we expect to have multiple implementations of this?

Reply:
Yes, there are multiple more transforms added in https://github.com/data-integrations/dlp-plugins/pull/5